network/vif-route-qubes

```diff
@@ -20,6 +20,7 @@
 #         this script).
 #============================================================================
 
+set -e
```

Comment:
I'm not sure if this only uncovered some pre-existing issue (more likely), or if there is a bug in this PR, but vif-qubes-nat.sh now fails. The end effect is that with this PR at least masquerading IP doesn't work anymore. The tests hanged, so I don't have full log, but I have a screenshot with some of the details (see the bottom part): https://openqa.qubes-os.org/tests/71340#step/system_tests/19

Another thing is most networking test fail with IPv6 enabled (https://openqa.qubes-os.org/tests/71351), but that could be  an issue with the tests themselves (given I found https://github.com/QubesOS/qubes-core-admin/pull/528 already). I'll analyze test logs a bit more.